db/db.js

```diff
@@ -0,0 +1,23 @@
+const Pool = require('pg').Pool;
+
+const pool = new Pool({
+    user: 'andersaustin', //needs to be changed to work in your instance
```

Comment:
Could you change local sensitive information into process.env.######? This should allow everyone to not have to change this everytime they pull down from github.